routes/server.go

```diff
@@ -1287,6 +1287,13 @@ func (fes *APIServer) NewRouter() *muxtrace.Router {
 			fes.UnregisterAsValidator,
 			PublicAccess,
 		},
+		{
+			"GetValidatorByPublicKeyBase58Check",
+			[]string{"GET", "OPTIONS"},
```

Comment:
```suggestion
			[]string{"GET"},
```
Do we need options here?